packages/arb-avm-cpp/data_storage/src/checkpointedmachine.cpp

```diff
@@ -181,10 +168,10 @@ void CheckpointedMachine::saveAssertion(const Assertion& assertion) {
         AggregatorStore::saveLog(*tx->transaction, logData);
     }
 
-    for (const auto& msg : assertion.outMessages) {
-        std::vector<unsigned char> msgData;
-        marshal_value(msg, msgData);
-        AggregatorStore::saveMessage(*tx->transaction, msgData);
+    for (const auto& send : assertion.sends) {
```

Comment:
I think we'll be changing `assertion.sends` be something like `std::vector<std::vector<char>>`

Reply:
change has bee completed on all sides